printer_mods/Tircown/Raspberry_plug_panel/readme.md

```diff
@@ -0,0 +1,8 @@
+# Voron 2.4 Raspberry vertically installed in plug panel
+
+For the DD11 version I used this reference: SCHURTER DD11.0113.1111
+Many other versions of SHURTER DDXX plugs are compatible. Check out sizes on datasheet.
+
```

Comment:
How do you mount the Pi bracket to the plug panel?  M3x8 + heatset inserts?  Can you add that to the README?